test/global_uid_test.rb

```diff
@@ -328,34 +328,6 @@ def table_exists?(connection, table)
     end
   end
 
-  describe "In dry-run mode" do
-    before do
-      GlobalUid::Base.global_uid_options[:dry_run] = true
-      CreateWithNoParams.up
-    end
-
-    it "increment normally1" do
-      (1..10).each do |i|
-        assert_equal i, WithGlobalUID.create!.id
-      end
-    end
-
-    it "insert into the UID servers nonetheless" do
-      GlobalUid::Base.expects(:get_uid_for_class).at_least(10)
-      10.times { WithGlobalUID.create! }
-    end
-
-    it "log the results" do
-      ActiveRecord::Base.logger.expects(:info).at_least(10)
-      10.times { WithGlobalUID.create! }
-    end
-
-    after do
-      reset_connections!
-      CreateWithNoParams.down
-    end
-  end
-
```

Comment:
Check also linee 444/416 of this filee.

Reply:
Ah, thanks for pointing that out. I had changed the defaults in my other PR in a later commit and missed it here. 💖